src/migrations/20191101053439-toggle-question-feedback.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
```

Comment:
Nit - this could be combined with the other migration to make the changes necessary for this PR completely atomic. See https://github.com/illinois/queue/blob/master/src/migrations/20190308215151-queue-admission-control.js for an example.